src/ResultSet.php

```diff
@@ -105,4 +113,11 @@ protected function yieldTraversable(Traversable $traversable)
             yield $key => $value;
         }
     }
+
+    public function disableChache()
```

Comment:
`disableCache`. Please also move it up directly under the constructor and add documentation.